app/src/main/java/edu/uc/groupproject/covid19tracker/dto/Cases.kt

```diff
@@ -2,7 +2,7 @@ package edu.uc.groupproject.covid19tracker.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Cases(@SerializedName("country_name") var country_name: ArrayList<String>,
+data class Cases(@SerializedName("country_name") var countryName: ArrayList<String>,
```

Comment:
Changed to better fit Kotlin naming conventions https://kotlinlang.org/docs/reference/coding-conventions.html